biorbd_optim/dynamics/problem.py

```diff
@@ -74,8 +74,8 @@ def torque_activations_driven_with_contact(ocp, nlp):
         """
         Problem.configure_q_qdot(nlp, True, False)
         Problem.configure_tau(nlp, False, True)
-        nlp["nbActuators"] = nlp["nbTau"]
-        if nlp["dynamics_type"].dynamics:
+        nlp.nbActuators = nlp.nbTau
+        if nlp.dynamics_type.dynamics:
```

Comment:
Please have a look if `dynamics_func` and `dynamics` are the same. If so, remove `nlp.dynamics_type.dynamics`

Reply:
It looks like these variables are not the same, I found three different places where they don't have the same value